### ISSUE TITLE
About the batchnorm layer.

### ISSUE THREAD

**Issue by fanrz** [cv-rits/LMSCNet#4]
Dear.
I find you did not use any norm layer.
Can you explain it?
usually, norm layer may help overfitting problem, but you did not use it, or consistent supervision (1/8, 1/4, 1/2 resolution) help?

**Comment by anhquancao** [comment]
Hi @fanrz, I discussed with the author about this and he said that he tried to add batchnorm but the performance is the same so he removed it.

**Comment by fanrz** [comment]
@anhquancao thanks.
I guess 1/8, 1/4, 1/2 resolution supervision may help, so batchnorm is not necessary, 
BN help make network more stable, consisitent supervision do the same thing. 
this is what I guess, I do not test and verfity。
if an end-to-end network with only full size supervision in the last, BN is common used.
Do you have any idea about it?

**Comment by anhquancao** [comment]
Hi @fanrz, as I remembered, the BN is not needed even with a single scale. 
In my opinion, the reason is that the input is an occupancy grid with either 0 or 1. Therefore, the distribution shift in the network is really small compared to an RGB image. Thus, the BN has almost no effect here.

**Comment by fanrz** [comment]
@anhquancao thanks for your reply.
